examples/feature_selection/plot_select_from_model_diabetes.py

```diff
@@ -143,3 +143,67 @@
 # attribute. The forward SFS is faster than the backward SFS because it only
 # needs to perform `n_features_to_select = 2` iterations, while the backward
 # SFS needs to perform `n_features - n_features_to_select = 8` iterations.
+
+# %%
+# Selecting features using Sequential Feature Selection and negative tolerance values
```

Comment:
This seems like a quite long title. My suggestion is to remove the title `Discussion` to keep the text as part of the narrative of the `Selecting features with Sequential Feature Selection` section. The this whole block can become a subsection called `Using negative tolerance values` or something similar.